internal/pkg/get/get.go

```diff
@@ -0,0 +1,77 @@
+package get
+
+import (
+	"fmt"
+
+	"github.com/jbrekelmans/kube-compose/pkg/config"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	clientV1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type getRunner struct {
+	cfg              *config.Config
+	k8sClientset     *kubernetes.Clientset
+	k8sServiceClient clientV1.ServiceInterface
+}
+
+type KubeComposeService struct {
+	Service   string
+	Hostname  string
+	Namespace string
+	ClusterIP string
+}
+
+func Service(cfg *config.Config, serviceName string) (KubeComposeService, error) {
```

Comment:
This name is not good, `GetServiceDetails` is better

Reply:
Refactored to ServiceDetails, as the package is already get so it ends up being get.ServiceDetails

Reply:
I don't like this

Reply:
`func name will be used as get.GetServiceDetails by other packages, and that stutters; consider calling this ServiceDetails (golint)`

Reply:
Linter also does not like this naming convention.

Reply:
Done